src/main/rx/spektrum.c

```diff
@@ -208,6 +208,11 @@ void spektrumBind(rxConfig_t *rxConfig)
 
 	LED1_ON;
 
+#ifdef STM32F10X
```

Comment:
Don't know if this is a good place for the definitions?
Maybe there is a better one?
Tried to put it in serial_uart_stm32f10x.c but this does not work.